Feat/#853 dashboard improvements

svetlana-b opened the pull request:
-   [x] Have you followed the [contributing guidelines](https://github.com/ever-co/gauzy/blob/master/.github/CONTRIBUTING.md)?
-   [x] Have you explained what your changes do, and why they add value?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

---

svetlana-b commented:
https://www.loom.com/share/88f2520cb6c64c16a8919ed4de6f1c80

evereq commented:
@svetlana-b few questions:
1) please check why you changed package.json file and added there ""@nestjs/cqrs": "^6.1.0"? (that's server-side package). Please make sure PR only changes files which needed :) (i.e. also package-lock.json probably should not be changed too)
2) I see you removed code which related to `proportion`. Why do you think it's not needed anymore? Why do you think someone added it before? How did feature work with it compared to without it, after your changes now? (every time we remove some code that someone else writes we need to make sure we improve everything without losing some existed important feature. So it's critically important to understand why such code was there before and how everything will work without it).

svetlana-b commented:
@evereq 1. I had issues with building after pull from develop, added "@nestjs/cqrs" package and removed package-lock.json file, then install yarn again. It solved the issues.
2. Found out why proportions to be added and changed Stacked Bar chart accordingly.

evereq commented:
@svetlana-b see my comments below please:

1. Please remove all changes in package.json and package-lock.json, they are not needed really and not sure how they helped you. Most probably they just trigger for you to reinstall some packages and that make sure all worked, not the fact that you added server-side package to client side project

2. Good that you added it (`proportion`) back, but I want to understand how it happens that you removed it? We need to be careful when our devs remove features which someone else added. Because that way we can go in the back direction: add 1 feature, remove 2, etc :) 

 Sure code reviews try to catch that, but as more code, you will write, the more hard and time consuming it's going to be to review it. So can you explain the reason why you removed some code? If no reason and you did it just by mistake, then just please try next time to double-check yourself when you removing code (especially large chunks of it)

3. In `theme.dark.ts` it looks like you replaced spaces with tabs (or back) and so it shows the whole file was changed, while you most probably just changed a few lines. Please talk with @AlexTasev or @rmagon or some other devs in our Slack to figure out how to prevent that in the future and if you can please fix that PR to only include changes in the code which you really changed.

svetlana-b commented:
@evereq 1. Remove changes from .json files
2. The code has been removed by mistake, will be more careful next time
3. I made no changes in theme.dark.ts, changed only edit-organization.component.scss and accounting.component.scss for fixing theme. The changes with spaces are made automatically with auto-format

svetlana-b commented:
Can't change theme.dark.ts file. Every time I try to change it back, auto-format made changes again

evereq commented:
@svetlana-b 
1) In https://github.com/ever-co/gauzy/pull/873/files I still see you change  package-lock.json

3) Please as I explained before, talk with other developers on how to make sure that autoformat working as it should so it does NOT make changes like that in the files anymore. And next try to revert your changes in theme file so it only contains what you changed, not formating changes!

After you will fix both 1) and 3) we will merge it

rmagon commented:
If you use visual studio code there is a "Detect indentation from content" option

<img width="617" alt="Screen Shot 2020-03-29 at 1 03 36 AM" src="https://user-images.githubusercontent.com/6750734/77832062-3eed7980-7159-11ea-86a8-5d0b680c190a.png">
